corehq/apps/app_manager/views/forms.py

```diff
@@ -251,7 +251,7 @@ def should_edit(attribute):
         if xform.exists():
             xform.set_name(name)
             save_xform(app, form, xform.render())
-        resp['update'] = {'.variable-form_name': form.name[lang]}
+        resp['update'] = {'.variable-form_name': trans(form.name, [lang], use_delim=False)}
```

Comment:
It looks like `trans(name, [lang], use_delim=False)` is essentially a more robust version of `name[lang]`, yeah?  I was confused about this change 'till I saw this bit from the source of `trans`:
```python
        if lang in name and name[lang]:
            return name[lang] + ("" if langs and lang == langs[0] else suffix(lang))
```